heartbeat/iSCSITarget.in

```diff
@@ -524,6 +551,23 @@ iSCSITarget_stop() {
 		ocf_release_lock_on_exit $TARGETLOCKFILE
 		ocf_run targetcli /iscsi delete ${OCF_RESKEY_iqn} || exit $OCF_ERR_GENERIC
 		;;
+	scst)
+		ocf_run scstadmin -disable_target ${OCF_RESKEY_iqn} -driver iscsi -force -noprompt
+		for i in $(find /sys/kernel/scst_tgt/targets/iscsi/${OCF_RESKEY_iqn}/ -name force_close); do
+			echo 1 > ${i}
+		done
+
+		timer=0
+		while ls -Ad /sys/kernel/scst_tgt/targets/iscsi/${OCF_RESKEY_iqn}/sessions/* > /dev/null 2>&1; do
+			if [ ${timer} -gt ${INIT_LOGOUT_TIMEOUT} ]; then
+				ocf_log warn "Some initiators still logged in after ${INIT_LOGOUT_TIMEOUT} seconds. Continuing."
+				break
+			fi
+			timer=$((timer + 1))
+			sleep 1
+		done
+
+		scstadmin -rem_target ${OCF_RESKEY_iqn} -driver iscsi -force -noprompt
```

Comment:
You should add ;; here as well, and after line 597.

Reply:
Argh, sorry, that should be all of them now.